src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work ⚙.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped its work.");

```

Comment:
Similar to the `doWork` method, the message in `stopWork` should be more informative and specify what the 'Bulldozer' was doing before it stopped.